src/main.py

```diff
@@ -28,13 +30,15 @@ def chatbot():
 
 
 def robot_tests(s=''):
-    run("atest/"+s)
+    run("atest/"+s, listener="atest/listeners/LoggerListener.py")
 
 def pytests():
     pytest.main(["utest/"])
 
 if __name__ == "__main__":
-    real_test_data_generator()
-    chatbot()
-    robot_tests()
-    pytests()
+    setup_logging(enabled=True, for_tests=False, console_logging=False ,file_logging=True)
+    
+    # real_test_data_generator()
```

Comment:
remove ?

Reply:
I use the main.py as a way of showcasing the functionality. This helps to show how each module can be used and run the tests. Also is used to quickly test stuff